Sig.App.Backend/Startup.cs

```diff
@@ -354,10 +357,16 @@ private static void AddDecoratedMailer(IServiceCollection services)
         {
             services.AddScoped<FluentMailer>();
 
-            services.AddScoped<IMailer>(x => new RetryingMailer(
+            services.AddScoped(x => new RetryingMailer(
                 x.GetService<FluentMailer>(),
                 x.GetService<ILogger<RetryingMailer>>())
             );
```

Comment:
Je pense que le RetryingMailer pourrait être ajouté simplement comme ça:
```suggestion
            services.AddScoped<RetryingMailer>();
```